src/api/test/functional/attributes_test.rb

```diff
@@ -306,12 +306,13 @@ def test_attrib_delete_permissions
     login_king
     data = "<attributes><attribute namespace='OBS' name='VeryImportantProject'/></attributes>"
     post '/source/home:tom/_attribute', params: data
+    assert_response :success
 
     login_tom
     delete '/source/home:tom/_attribute/OBS:VeryImportantProject'
     assert_response 403
     delete '/source/home:tom/_attribute/?namespace=OBS&name=VeryImportantProject'
-    assert_response 403
+    assert_response 404
```

Comment:
Why is this a 404 now when it was not removed in the api call before?

Reply:
because that route no longer exists.

Reply:
I considered removing it - but I thought there is value in asserting you can't delete the attribute through the old route. But I'll do as you say.

Reply:
See 97389774297e5426b2a20fa56d5ec39dece4c80c